components/installer/pkg/config/v1/config.go

```diff
@@ -227,6 +227,13 @@ type Workspace struct {
 	Templates *WorkspaceTemplates `json:"templates,omitempty"`
 }
 
+type LicensorType string
+
+const (
+	LicensorTypeLegacy     LicensorType = "legacy"
```

Comment:
let's not name this `legacy` as it implies that we don't want to use this anymore in all cases, and that it's likely to go away.

That's not true - it's just a different means. How about `gitpod-signed`?